common/parse_parallelization_setup.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+
+from ._parsing import _read_results_lines
+
+
+def parse_parallelization_setup(filename: Path):
```

Comment:
I imagine that this code is just my prototype code, which should not be considered to be correct.  If this is true, please indicate this and the need to review this code with a TODO statement.

Same for `parse_performance_results` and any other functions that you and Manuel have not explicitly taken ownership of.